Ajout Seringue du Berserker:

medzila opened the pull request:
+L'utilisateur inflige 1.5x plus de degats
+Il recoit 1.25x plus de degats
+Coef à recalibrer au cas ou

medzila committed:
Ajout Seringue du Berserker:
+L'utilisateur inflige 1.5x plus de degats
+Il recoit 1.25x plus de degats
+Coef à recalibrer au cas ou

medzila commented:
je sais pas si c'est comme ca que ca marche :(

kossolax commented:
Je crois qu'il serai plus sage de modifier l'enum des "chirurgies" que celui en amont. Le core ne peut pas gérer ça sans une recompilation.
+ Il n'y a pas besoin de créer un nouveau forward. On peut gérer ça via fwdChiruForce.

 L'item va poser problème si le mec essaye de l'utiliser une 2ème fois.
Penser à ajouter un effet? Sur la vue ? Colorier le joueur? Un halo autour de lui ?

kossolax commented:
Après réflexion... Je pense que fusionner adrénaline et berzek pourrait être très intéressant pour le job.

medzila committed:
Ajout Seringue du Berserker
+b_Berserk enlevé & it_Berserk ajouté a la place dans l'enum chiru
+fwdBerserk enlevé & ajout du body dans le fwdChiru 200% de degats
+halo rouge autour de l'utilisateur
+A rajouter= degats decroissants

medzila commented:
J'ai corrigé ce que tu m'as dis, manque les degats décroissants.
Peut etre rajouter du CD si l'item est trop OP?

kossolax commented:
A cause du changement dans l'API impossible de merge, y a conflit chef.

medzila commented:
Je m'en occupe !

medzila committed:
Modif Seringue Berserker (API 0.14)
+Utilisation du PreGiveDamage
+Les degats recus et infligés décroient au cours du temps

medzila commented:
J'ai modifié suivant l'API 0.14 dis moi si ca te semble correct :[

kossolax commented:
Hummm, tu te prends encore la tête sur le truc. Comment t'as fait pour règler le problème? T'as copier coller les fichiers? le diff est chelou. @Hellhium
